docs/topics/proc_running-the-secured-booster-integration-tests-on-vx.adoc

```diff
@@ -0,0 +1,30 @@
+[#running-the-secured-booster-integration-tests_context]
+= Running the {name-mission-secured} Booster Integration Tests
+
+.Prerequisites
+
+* The `oc` client authenticated.
+// * An empty OpenShift project in which to run the tests.
+
+.Procedure
+// space here to prevent the section title and admonition from sticking together in the rendered version
+{nbsp}
+include::note-integration-tests-app-deletion-warning.adoc[]
+
+By default, the SSO server is deployed (and destroyed) as part of testing. The steps for executing integration tests are as follows:
+
+. In a terminal application, navigate to the directory with your project.
+. Execute the integration tests:
++
+[source,bash,options="nowrap",subs="attributes+"]
+--
+mvn clean verify -Popenshift,openshift-it
+--
+
+If you deployed an SSO server beforehand, e.g. by executing `oc create -f service.sso.yaml`, set the system property `skip.sso.init` to `true` when running the tests (example shown below).
```

Comment:
Parens are a big no-no in our docs :-)

We can do something like this, though:

```
If you deployed an SSO server beforehand, e.g. by executing `oc create -f service.sso.yaml`, set the system property `skip.sso.init` to `true` when running the tests:

--
mvn clean verify -Popenshift,openshift-it -Dskip.sso.init=true
--

When executed like this, the tests will use the existing SSO server.
The tests will not deploy their own SSO server, nor will they destroy the existing one.
```

WDYT?

Reply:
Sounds good, I'll update it right away if no one has any objections :)